### PR TITLE
Nas 104272

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html [hunk 1]
```diff
@@ -23,4 +23,7 @@
     <mat-error *ngIf="control.hasError('regex')">
         {{"Invalid format or character." | translate}}
     </mat-error>
+    <mat-error *ngIf="control.hasError('ip')">
+        {{"Select " | translate}} {{control.errors.info[1]}} {{" to include all " | translate}} {{control.errors.info[0] | translate}} {{" addresses. When this has been chosen, additional addresses cannot be selected." | translate}}
```
Comment (erinix) [inline src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html]: @freenas/docs-team  could you review this string?

**Comment by erinix** [comment]
looks good

File: src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html (hunk 1 above)
Comment (wblock) [reply src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html]: control.errors.info seems like a misleading variable name.  It is "All IPv4" or "All IPv6" or "All IPv4 and IPv6"?  The rest of the string seems reasonable.

Comment (lola27) [reply src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html]: > control.errors.info seems like a misleading variable name. It is "All IPv4" or "All IPv6" or "All IPv4 and IPv6"? The rest of the string seems reasonable.

control.errors.info include three items like this [ipType === 'ipv4' ? 'IPv4' : 'IPv6', wildcard, thisControl.value[i]], which means the ip type, the value which means all ipv4 or ipv6 (0.0.0.0 or ::), the current field value.

**Pull request merged**